doc/source/index.rst

```diff
@@ -0,0 +1,333 @@
+message to trigger their role and will send another posttroll message when
+their task is done.
+
+.. _cat:
+
+cat
+^^^
+
+Concatenates granules or segments to a single file.  This may be a useful step
+before using external software for preprocessing data.  You will need `Kai`_
+from EUMETSAT to concatenate MetOp AVHRR granules.  Cat listens to input
+messages via posttroll according to topics defined in the configuration file.
+Upon completion, it will publish a posttroll message with topic defined in
+the configuration file.
+
+The kai configuration file is in the INI format and should have one ore
+more sections.  Each section may have the following fields:
+
+output_file_pattern
+    A pattern (trollsift syntax) on what the output file should be.
+
+aliases
+    Optional (what does it do?)
+
+min_length
+    Optional, integer, minimum number of minutes needed to consider the data
+
+command
+    Command used for concatenation.
+
+stdout
+    Optional; if command writes to stdout, redirect output here.
+
+publish_topic
+    Optional; publish a message when file is produced, using this topic.
+
+publish_port
+    Optional; use a custom port when publishing a message.
+
+nameservers
+    Optional; nameservers to publish on.
+
+subscriber_nameserver
+    Optional; nameserver to listen to.
+
+.. _Kai: https://navigator.eumetsat.int/product/EO:EUM:SW:METOP:165
+
+Example configuration:
+
+.. literalinclude:: ../../examples/cat.cfg_template
+   :language: ini
+
+catter
+^^^^^^
+
+Alternative to cat that does something else.
+
+.. literalinclude:: ../../examples/catter.cfg_template
+   :language: ini
+
+.. _gatherer:
+
+gatherer
+^^^^^^^^
+
+Watches files or messages and gathers satellite granules in "collections",
+sending then the collection of files in a message for further processing.
+Determines what granules with different start times belong together.
+A use case may be a reception system in which a single overpass results in
+multiple files, that should be grouped together for further processing.
+It uses `pytroll-schedule`_ to estimate the area coverage based on start
+and end times contained in filenames.
+
+The  ``gatherer`` collection is started when it receives a posttroll message,
+perhaps from `trollstalker`_ or `segment-gatherer`_.  Using the configured
+granule duration and the area of interest, it calculates the starting times
+of granules it should expect to be covered in this area before and after the
+granule it was messaged about.  Collection is considered finished when either
+all expected granules have been collected or when a timeout is reached,
+whatever comes first.  Timeout is configured with the ``timeliness`` option
+(see below).
+
+.. _pytroll-schedule: http://pytroll-schedule.readthedocs.org/
+
+The configuration file in INI format needs a section called ``[DEFAULT]``
+and one or more sections
+corresponding to what should be gathered.  The ``[DEFAULT]`` section should contain
+exactly one field:
```

Comment:
No necesasrily `exactly one`. The `[DEFAULT]` section is a special thing in `.ini` format, and it holds the common items for all the other sections. AFAIK the values in `[DEFAULT]` can be overridden in the normal sections.

Reply:
Adapted description.